docs/date.md

```diff
@@ -80,6 +75,14 @@ date === Temporal.Date.from(date)  // => true
 date = Temporal.Date.from({year: 2006, month: 8, day: 24});  // => 2006-08-24
 date = Temporal.Date.from(Temporal.DateTime.from('2006-08-24T15:43:27'));
   // => same as above; Temporal.DateTime has year, month, and day properties
+
+// Different disambiguation modes
+date = Temporal.Date.from({ year: 2001, month: 13, day: 1 }, 'constrain')  // => 2001-12-01
+date = Temporal.Date.from({ year: 2001, month: -1, day: 1 }, 'constrain')  // => 2001-01-01
+date = Temporal.Date.from({ year: 2001, month: 13, day: 1 }, 'balance')  // => 2002-01-01
+date = Temporal.Date.from({ year: 2001, month: -1, day: 1 }, 'balance')  // => 2000-11-01
```

Comment:
This one doesn't make sense to me, wouldn't the "next highest unit" be "2001-01-01" ? Why does it go to november 2000?

Reply:
Here's how I'd interpret it: month 0 would be one month before 2001-01, which would be December 2000, so month –1 would be two months before 2001-01, so November 2000.

Reply:
I see @ptomato thanks

Reply:
I also didn't get this immediately, so maybe add the zero case to the docs as well.